guides/common/modules/proc_setting-an-alternative-directory-for-remote-execution-jobs-in-pull-mode.adoc

```diff
@@ -0,0 +1,33 @@
+[id="setting-an-alternative-directory-for-remote-execution-jobs-in-pull-mode_{context}"]
+= Setting an alternative directory for remote execution jobs in pull mode
+
+By default, {Project} uses the `/run` directory on host for remote execution jobs in pull mode.
+
+If the `/run` directory on your host is mounted with the `noexec` flag, {Project} cannot execute remote execution job scripts in this directory.
+Use this procedure set a new directory for executing remote execution jobs in pull mode.
+
+.Procedure
+. On your host, create a new directory:
```

Comment:
That ^ said, we have a [BZ#2258870](https://bugzilla.redhat.com/show_bug.cgi?id=2258870) that asks to specify that the steps are to be performed on SmartProxy as well. Would this make sense?

```suggestion
On {ProjectServer} or any other {SmartProxy}, perform these steps:

. Create a new directory:
```

Reply:
The comment in the BZ only applies to push-mode REX jobs. As for the BZ itself, I believe you can close it as completed once this PR is merged and the docs are republished.